src/client/src/index.tsx

```diff
@@ -39,7 +39,7 @@ async function init() {
     // If you want your app to work offline and load faster, you can change
     // unregister() to register() below. Note this comes with some pitfalls.
     // Learn more about service workers: https://bit.ly/CRA-PWA
-    serviceWorker.register()
+    serviceWorker.unregister()
```

Comment:
The service worker is a nice optimization but it does add another layer of complexity to the caching policy. In my limited tests it appeared that it cached the `index.html` file which is bad as it would never get updated. So instead of digging into the configuration I chose to remove it as we have a small number of assets to cache.

Reply:
since it is disabled, should we remove this code completely?

Reply:
The call to `unregister()` will remove the service worker from any clients that have it installed.